packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -34,15 +41,19 @@ export async function prepareRepoData(
 ) {
     try {
         const files = await collectFiles(repoRootPaths, workspaceFolders, true, maxRepoSizeBytes)
+        const devCommandWorkspaceConfigurations = CodeWhispererSettings.instance.getDevCommandWorkspaceConfigurations()
+        const useAutoBuildFeature = devCommandWorkspaceConfigurations[repoRootPaths[0]] ?? false
```

Comment:
are the test for prepare repo data updated to handle this?

Reply:
are you mentioning `prepareRepoData.test.ts`?

Reply:
yeah

Reply:
they seem to be passing so i guess its fine